modules/luci-base/htdocs/luci-static/resources/network.js

```diff
@@ -3374,7 +3374,7 @@ WifiDevice = baseclass.extend(/** @lends LuCI.network.WifiDevice.prototype */ {
 	 */
 	getHWModes: function() {
 		var hwmodes = this.ubus('dev', 'iwinfo', 'hwmodes');
-		return Array.isArray(hwmodes) ? hwmodes : [ 'b', 'g' ];
+		return Array.isArray(hwmodes) ? hwmodes : [ 'b', 'g', 'n', 'ac', 'ax', 'be' ];
```

Comment:
I'm a bit unsure about this change, what I assume happens here is that if the HW modes for a specific wireless device is returned as an array, it will use that one as the return value of `getHWModes`, and if not, it falls back to the array. 

With this change this now seems to indicate HW mode capabilities up to 802.11be for _all_ wireless devices if they don't return a proper HW modes array. I _assume_ the default is ['b', 'g'] since when this code was introduced, this was the supported 802.11 stacks in OpenWrt. Has this been tested on non-WiFi 7-devices and the eventual impact it might have?

Reply:
so I tested this on ip807x qualcommax (qnap) - as you can see BE doesn't show based on the iwinfo capabilities - so what I would do is to put it back to the original ... the banana pi4 still shows 'be'
![image](https://github.com/user-attachments/assets/c7bb660d-0d2a-431d-9ff6-8b627f152bbf)

let me know if you agree - probably worth adding a comment to that function on the likes of what you just made

Reply:
Sounds like a good idea, feel free to add some documentation in the code if you feel it would help.